File: PepcodingSept_19/Lec_002(factorialNCRAndRotate)/ncr.java
import java.util.Scanner;
public class ncr
{
 public static Scanner scn=new Scanner(System.in);

 public static void main(String[] args)
 {
     System.out.print("Enter n for Factorial :");
     int n=scn.nextInt();
     System.out.println(Facto(n));
     nc_r();

 }
 public static void nc_r()
  {
    System.out.println("Enter N:");
    int n=scn.nextInt();
    System.out.println("Enter R:");
    int r=scn.nextInt();
    System.out.println((Facto(n))/(Facto(n-r)*Facto(r)));

  }
 public static int Facto(int n)
 {
     int ans=1;
     for(int i=1;i<=n;i++)
      {
         ans*=i;
      }
      return ans;
 }
}

File: PepcodingSept_19/Lec_002(factorialNCRAndRotate)/ncr.cpp
#include<iostream>

using namespace std;
int Facto(int n)
{
    int ans=1;
    for(int i=1;i<=n;i++)
     {
         ans*=i;
     }
     return ans;
}
void nc_r()
{
    int n,r;
    cout<<"Enter N:";
    cin>>n;
    cout<<endl;
    cout<<"Enter R:";
    cin>>r;
    cout<<endl;
    cout<<(Facto(n))/(Facto(n-r)*Facto(r));
}
int main(int args,char** argv)
{
    nc_r();
    return 0;
}